easybuild/easyconfigs/e/EMAN2/EMAN2-2.2-gompi-1.7.20.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock='Binary'
+
+name = "EMAN2"
+version = "2.2"
+
+homepage = 'http://blake.bcm.edu/emanwiki/EMAN2'
+description = """EMAN2 is the successor to EMAN1. It is a broadly based greyscale scientific image 
+ processing suite with a primary focus on processing data from transmission electron microscopes"""
+
+toolchain = {'name': 'gompi', 'version': '1.7.20'}
```

Comment:
@boegel some code uses mpi. The binary installer includes a mpi library but if you need to use it in the cluster you should relink those apps with the mpi library available in the cluster. That's why I use gompi and I use postinstallcmds. Check details here http://blake.bcm.edu/emanwiki/EMAN2/Install/BinaryInstallAnaconda#Linux_Clusters